modules.json

```diff
@@ -179,41 +179,6 @@
             },
             "subworkflows": {
                 "nf-core": {
-                    "bam_markduplicates_picard": {
-                        "branch": "master",
-                        "git_sha": "0c38be7e652a0b2f3a37681ee4c0dbdf85677647",
-                        "installed_by": ["subworkflows"]
-                    },
-                    "bam_sort_stats_samtools": {
-                        "branch": "master",
-                        "git_sha": "0c38be7e652a0b2f3a37681ee4c0dbdf85677647",
-                        "installed_by": ["fastq_align_bowtie2", "fastq_align_bwa", "fastq_align_chromap"]
-                    },
-                    "bam_stats_samtools": {
-                        "branch": "master",
-                        "git_sha": "0c38be7e652a0b2f3a37681ee4c0dbdf85677647",
-                        "installed_by": ["bam_markduplicates_picard", "bam_sort_stats_samtools"]
-                    },
-                    "fastq_align_bowtie2": {
-                        "branch": "master",
-                        "git_sha": "cfd937a668919d948f6fcbf4218e79de50c2f36f",
-                        "installed_by": ["subworkflows"]
-                    },
-                    "fastq_align_bwa": {
-                        "branch": "master",
-                        "git_sha": "cfd937a668919d948f6fcbf4218e79de50c2f36f",
-                        "installed_by": ["subworkflows"]
-                    },
-                    "fastq_align_chromap": {
-                        "branch": "master",
-                        "git_sha": "cfd937a668919d948f6fcbf4218e79de50c2f36f",
-                        "installed_by": ["subworkflows"]
-                    },
-                    "fastq_fastqc_umitools_trimgalore": {
-                        "branch": "master",
-                        "git_sha": "cfd937a668919d948f6fcbf4218e79de50c2f36f",
-                        "installed_by": ["subworkflows"]
-                    },
```

Comment:
some subworkflows got deleted here, is that expected?

Reply:
Good spot! Probably, I deleted them by mistake when resolving the merge conflicts, I'll take a look